src/utils/deleteQueryParam/deleteQueryParamsByName.ts

```diff
@@ -1,8 +1,5 @@
-const deleteQueryParamsByName = (
-  keysToRemove: string[],
-  query: { [key: string]: string | string[] | undefined }
-): { [key: string]: string | string[] | undefined } => {
-  keysToRemove.forEach((key: string) => delete query[key])
+const deleteQueryParamsByName = (keysToRemove: string[], query: URLSearchParams): URLSearchParams => {
```

Comment:
I think to improve clarity we could rename this function to `deleteQueryParamsByKey` and the other one `deleteQueryParamsByKeyValue`, what do you think?

Reply:
Done